src/v2/services/implementations/DappStakingService.ts

```diff
@@ -49,12 +49,39 @@ export class DappStakingService implements IDappStakingService {
     return new TvlModel(tvl, tvlDefaultUnit, tvlUsd);
   }
 
+  public async nominationTransfer({
+    amount,
+    fromContractId,
+    targetContractId,
+    address,
+  }: {
+    amount: BN;
+    fromContractId: string;
+    targetContractId: string;
+    address: string;
+  }): Promise<void> {
+    Guard.ThrowIfUndefined('fromContractId', fromContractId);
+    Guard.ThrowIfUndefined('targetContractId', targetContractId);
+    Guard.ThrowIfUndefined('stakerAddress', address);
+
+    const stakeCall = await this.dappStakingRepository.getNominationTransferCall({
+      amount,
+      fromContractId,
+      targetContractId,
+    });
+    await this.wallet.signAndSend(
+      stakeCall,
+      address,
+      `You successfully staked to ${targetContractId} from ${fromContractId}`
```

Comment:
Maybe to put this to translation file. I also always forget to do that.

Reply:
@bobo-k2  True, but I'm wondering how we add `useI18n()` to Vuex files. Do you have any idea?

Reply:
Otherwise, we can create an argument like `successMsg` for this function but I'd like to avoid this approach if we have another way to fix this.

Reply:
Yes, you are right. Now I remember I have the same issue. Let's leave as is for now